extensions/big-data-cluster/src/bigDataCluster/controller/clusterControllerApi.ts

```diff
@@ -174,24 +174,17 @@ export class ClusterController implements IClusterController {
 	}
 
 	public async getKnoxUsername(sqlLogin: string): Promise<string> {
-		try {
-			// This all is necessary because prior to CU5 BDC deployments all had the same default username for
-			// accessing the Knox gateway. But in the allowRunAsRoot setting was added and defaulted to false - so
-			// if that exists and is false then we use the username instead.
-			// Note that the SQL username may not necessarily be correct here either - but currently this is what
-			// we're requiring to run Notebooks in a BDC
-			const config = await this.getClusterConfig();
-			return config.spec?.spec?.security?.allowRunAsRoot === false ? sqlLogin : DEFAULT_KNOX_USERNAME;
-		} catch (err) {
-			console.log(`Unexpected error fetching cluster config for getKnoxUsername ${err}`);
-			// Optimistically fall back to SQL login since root shouldn't be typically used going forward
-			return sqlLogin;
-		}
-
+		// This all is necessary because prior to CU5 BDC deployments all had the same default username for
+		// accessing the Knox gateway. But in the allowRunAsRoot setting was added and defaulted to false - so
+		// if that exists and is false then we use the username instead.
+		// Note that the SQL username may not necessarily be correct here either - but currently this is what
+		// we're requiring to run Notebooks in a BDC
+		const config = await this.getClusterConfig();
+		return config.spec?.spec?.security?.allowRunAsRoot === false ? sqlLogin : DEFAULT_KNOX_USERNAME;
 	}
 
 	public async getClusterConfig(promptConnect: boolean = false): Promise<any> {
-		return await this.withConnectRetry<IEndPointsResponse>(
+		return await this.withConnectRetry<any>(
```

Comment:
Is there a more specific type we can return for this method?

Reply:
Sorta - the config isn't a hard definition but we could add in some of the groups and well-known properties. I'll probably be looking at that when we start work on the config changes planned for later though. For now I was just fixing this since it wasn't using the right type here (it returns an any - the generic type was probably just a copy paste error)